### PR TITLE
en: update safe mode description

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add some description about "safe mode".

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/444
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/glossary.md [hunk 1]
```diff
@@ -87,15 +87,19 @@ The relay processing unit is the processing unit used in DM-worker to pull binlo
...
 
 ## S
 
 ### Safe mode
 
-Safe mode is the mode in which DML statements can be imported more than once when the primary key or unique index exists in the table schema.
+Safe mode is the mode in which DML statements can be imported more than once when the primary key or unique index exists in the table schema. In this mode, some statements from the upstream are migrated to the downstream only after they are re-written. The `INSERT` statement is re-written as `REPLACE`; the `UPDATE` statement is re-written as `DELETE` and `REPLACE`.
 
-In this mode, some statements from the upstream are migrated to the downstream only after they are re-written. The `INSERT` statement is re-written as `REPLACE`; the `UPDATE` statement is re-written as `DELETE` and `REPLACE`. TiDB DM automatically enables the safe mode within 5 minutes after the migration task is started or resumed. You can manually enable the mode by modifying the `safe-mode` parameter in the task configuration file.
+This mode is enabled in the follow situations:
```
Comment (yikeke) [inline en/glossary.md:98]:
```suggestion
This mode is enabled in any of the following situations:
```

File: en/glossary.md [hunk 2]
```diff
@@ -87,15 +87,19 @@ The relay processing unit is the processing unit used in DM-worker to pull binlo
...
+
+- TiDB DM automatically enables the safe mode for 5 minutes after the incremental replication task is started or resumed.
```
Comment (yikeke) [inline en/glossary.md:100]:
```suggestion
- TiDB DM automatically enables the safe mode within 5 minutes immediately after the incremental replication task is started or resumed.
```

File: en/glossary.md [hunk 3]
```diff
@@ -87,15 +87,19 @@ The relay processing unit is the processing unit used in DM-worker to pull binlo
...
+- The safe mode remains enabled when the `safe-mode` parameter in the task configuration file is set to `true`.
+- If the argument `--consistency none` is configured in the Dumpling processing unit of a full migration task, it cannot be determined whether the changes of binlog files happened at the beginning of the export affect the exported data or not. Therefore, the safe mode remains enabled for the incremental replication of these binlog files.
```
Comment (yikeke) [inline en/glossary.md:102]:
```suggestion
- If the argument `--consistency none` is configured for the dump processing unit of a full migration task, it cannot be determined whether the binlog changes at the beginning of the export affect the exported data or not. Therefore, the safe mode remains enabled for the incremental replication of these binlog files.
```

**Comment by yikeke** [comment]
Rest LGTM

File: en/glossary.md (hunk 3 above)
Comment (lance6716) [reply en/glossary.md:102]: last words "binlog files" could be "binlog changes" or "binlog positions". A file is too large, containing many binglog positions.

**Review by lance6716 — COMMENTED** [review]
rest LGTM

File: en/glossary.md (hunk 3 above)
Comment (CharLotteiu) [reply en/glossary.md:102]: In en/task-configuration-file-full.md, this processing unit is called Dumpling processing unit. Also see https://github.com/pingcap/docs-dm/pull/430/files.

Comment (yikeke) [reply en/glossary.md:102]: Refer to [the glossary](https://docs.pingcap.com/tidb-data-migration/v2.0/glossary#dump-processing-unit). The usage should be consistent and "dump processing unit".

Comment (yikeke) [reply en/glossary.md:102]: Can you help update the inconsistent usage? @CharLotteiu

Comment (CharLotteiu) [reply en/glossary.md:102]: Sure. 💪🏻

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**